staging/src/k8s.io/component-base/logs/json/klog_test.go

```diff
@@ -116,6 +118,13 @@ func TestKlogIntegration(t *testing.T) {
 			},
 			format: `{"ts":%f,"caller":"json/klog_test.go:%d","msg":"test","v":0,"count":1}`,
 		},
+		{
+			name: "KObj",
```

Comment:
Can you also add a test for KObjs?

Reply:
Not yet - https://github.com/kubernetes/kubernetes/pull/104103 needs to be merged first.

But I've tested locally based on that PR and KObjs prints the slice as structs:
```
		{
			name: "KObjs",
			fun: func() {
				klog.InfoS("several", "pods",
					klog.KObjs([]interface{}{
						&kmeta{Name: "pod-1", Namespace: "kube-system"},
						&kmeta{Name: "pod-2", Namespace: "kube-system"},
					}))
			},
			format: `{"ts":%f,"caller":"json/klog_test.go:%d","msg":"several","v":0,"pods":[{"name":"pod-1","namespace":"kube-system"},{"name":"pod-2","namespace":"kube-system"}]}`,
		},
```

Reply:
heh, that makes sense as slice doesn't implement Stringer.